jenkins-schema.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash -xe
 export DISPLAY=:99
-export GOVUK_APP_DOMAIN=test.alphagov.co.uk
-export GOVUK_ASSET_ROOT=http://static.test.alphagov.co.uk
+export GOVUK_APP_DOMAIN=test.gov.uk
```

Comment:
Can you remove these, and put them in `spec_helper`? There's no reason tests on CI should have different env variables for these from development.